src/homework2/task4.py

```diff
@@ -0,0 +1,34 @@
+"""Посчитать количество строчных (маленьких) и прописных (больших) букв в
+    введенной строке. Учитывать только английские буквы.
+"""
+
+
+def count_letters(str_):
+    """Подсчет символов.
+
+    :param str_: входная строка
+    :return: кортеж. (low_number, up_number). low_number - количество строчных,
+                                              up_number - количество пописных.
+    """
+
+    # write your code here
+
+
+s = input()

```

Comment:
please put your solution in function. in fact this file wouldn't be run because of indentation error